rcl/src/rcl/security_directory.c

```diff
@@ -247,12 +247,12 @@ char * rcl_get_secure_root(
     // Check node_secure_root is not NULL before checking directory
     if (NULL == node_secure_root) {
       RCL_SET_ERROR_MSG_WITH_FORMAT_STRING(
-        "SECURITY ERROR: unable to find a folder matching the node name in %s%s."
-        "Lookup strategy: %s",
-        ros_secure_root_env, node_namespace, lookup_strategy);
+        "SECURITY ERROR: unable to find a folder matching the node name '%s' in '%s%s'.\n"
+        "Lookup strategy: '%s'",
```

Comment:
Since the `lookup_strategy` is already separated with the colon we usually don't quote it.

Same below.